modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterInitializer.java

```diff
@@ -121,12 +121,12 @@ public CompletableFuture<Void> initCluster(
                                 e = e.getCause();
                             }
 
-                            LOG.error("Initialization failed: {}", e, e.getMessage());
+                            LOG.info("Initialization failed [reason={}]", e, e.getMessage());
```

Comment:
why you changed error leve to info?

Reply:
Error is always actionable. At this particular case the exception is returned back to the invoker, so it's up to invoker to decide whether the exception is recoverable or not. If exception is recoverable, then there is no need to bother operation team.